spec/system/candidate_entering_contact_details_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+describe 'A candidate entering contact details' do
+  include TestHelpers::PersonalDetails
```

Comment:
Does this spec need this helper?

Reply:
It does! (I'm blind).

These specs should get a little better when we've split out into separate models, as we won't have to fill out the entire form to test one part of it.